Desktop/components/JASP/Widgets/FileMenu/PrefsUI.qml

```diff
@@ -191,6 +191,23 @@ ScrollView
 				KeyNavigation.down:		uiScaleSpinBox
 			}
 
+			Text
+			{
+                id:                     translationDocLink
+
+				text:			qsTr("Help us translate or improve JASP in your language.")
```

Comment:
I preferred your text from https://github.com/jasp-stats/jasp-desktop/pull/4625#issuecomment-919948474. `or improve JASP in your language` sounds a little odd to me. One possibility to shorten it:

```
If you would like to improve our translations please visit our website.
```
Having only one part of the sentence be a hyperlink instead of the whole thing also helps set it apart more IMO

Reply:
I know what you mean, but I think the current sentence is more suitable for the UI. It's more actionable, "Help us ...". In addition, having the second QML element adds extra complication for later translation and alignment. That's why I changed it to be a single sentence.